app/messages/en.json

```diff
@@ -83,7 +83,7 @@
   "components_common_interactionButton_feedback": "Feedback",
   "components_common_interactionButton_logIn": "Log in",
   "components_common_interactionButton_clear": "Remove",
-  "components_common_interactionButton_addUpdate": "Add post",
```

Comment:
Here "Post" is also ok (same as Beitrag above)

Reply:
fixed all lines in new commit